frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -86,12 +89,12 @@ export const dashboardLogic = kea({
         ],
     }),
 
-    selectors: ({ props, selectors }) => ({
-        items: [() => [selectors.allItems], allItems => allItems.filter(i => !i.deleted)],
+    selectors: ({ props, selectors, values }) => ({
+        items: [() => [selectors.allItems], allItems => allItems?.items?.filter(i => !i.deleted)],
         itemsLoading: [() => [selectors.allItemsLoading], allItemsLoading => allItemsLoading],
         dashboard: [
-            () => [dashboardsModel.selectors.dashboards],
-            dashboards => dashboards.find(d => d.id === props.id) || null,
+            () => [selectors.allItems, dashboardsModel.selectors.dashboards],
+            dashboards => (values.allItems ? values.allItems : dashboards.find(d => d.id === props.id) || null),
```

Comment:
1) You shouldn't use `values` when creating a selector. 
2) This looks broken. The selector `selectors.allItems` gets converted to a param `dashboards` which is assumed to be an array... (`dashboards.find(...)`)? 

I guess you should just prepend `allItems` as the first argument to the function.

Reply:
Weirdly that did work! But changed to selectors.

It's slightly annoying now as "allItems" is basically the dashboard with items loaded into it, but the code is (I think!) correct and works.